### PR TITLE
Add AccountUtils.wipe() method instead of localstore.clear()

### PR DESCRIPTION
**Pull request by Daniel-Wang**
## Priority
Urgent

## Description
* Refactor clearing data code into an AccountUtils method, instead of using LocalStore.clear

### CODE REVIEW

File: app/src/main/java/blockeq/com/stellarwallet/activities/BaseActivity.kt [hunk 1]
```diff
@@ -17,7 +18,7 @@ abstract class BaseActivity : AppCompatActivity() {
             if (!WalletApplication.localStore.encryptedPhrase.isNullOrEmpty() && !WalletApplication.localStore.stellarAccountId.isNullOrEmpty()) {
                 launchPINView(PinType.LOGIN, "", "", null, true)
             } else {
-                WalletApplication.localStore.clearUserData()
+                AccountUtils.wipe(this, WalletApplication.userSession.pin)
```
Comment (dabitdev) [inline app/src/main/java/blockeq/com/stellarwallet/activities/BaseActivity.kt]: do we have the pin at this point?

Comment (Daniel-Wang) [reply app/src/main/java/blockeq/com/stellarwallet/activities/BaseActivity.kt]: Oh actually, we never have it at this point. Which means we can't clear the alias

Comment (Daniel-Wang) [reply app/src/main/java/blockeq/com/stellarwallet/activities/BaseActivity.kt]: But in the other situation in PinActivity, we can clear it there

Comment (dabitdev) [reply app/src/main/java/blockeq/com/stellarwallet/activities/BaseActivity.kt]: I would not make the wiping depending on the pin.
To clear the keystore I would get aliases() and delete one by one.
```
   List<String> list =  keyStore.aliases()
   for(String alias : list ) {
     keyStore.deleteEntry(alias)
  }
```

Comment (Daniel-Wang) [reply app/src/main/java/blockeq/com/stellarwallet/activities/BaseActivity.kt]: Okay that sounds good

File: app/src/main/java/blockeq/com/stellarwallet/encryption/KeyStoreWrapper.kt [hunk 2]
```diff
@@ -64,6 +64,16 @@ class KeyStoreWrapper(private val context: Context) {
      */
     fun removeAndroidKeyStoreKey(alias: String) = keyStore.deleteEntry(alias)
 
+    /**
+     * Clears all aliases in the Android Key Store
+     */
+    fun clear() {
```
Comment (dabitdev) [inline app/src/main/java/blockeq/com/stellarwallet/encryption/KeyStoreWrapper.kt]: could you write a small unit test for this?

Comment (Daniel-Wang) [reply app/src/main/java/blockeq/com/stellarwallet/encryption/KeyStoreWrapper.kt]: Yup!

File: app/src/androidTest/java/blockeq/com/stellarwallet/encryption/KeyStoreWrapperTest.kt [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            "access",
+            "accident")
+
+    @Test
+    fun clear_aliases() {
+        val context = InstrumentationRegistry.getTargetContext()
+        val keyStoreWrapper = KeyStoreWrapper(context)
+
+        aliases.forEach {
+            AccountUtils.getEncryptedMnemonicPhrase(context, mnemonic.joinToString(" "), null, it)
```
Comment (dabitdev) [inline app/src/androidTest/java/blockeq/com/stellarwallet/encryption/KeyStoreWrapperTest.kt]: it is a good test, but you should focus on the CUT (class under test)
`AccounUtils` should have his own test class.
Could you create the aliases using `KeystoreWrapper.createAndroidKeyStoreAsymmetricKey(pin)`

Comment (dabitdev) [reply app/src/androidTest/java/blockeq/com/stellarwallet/encryption/KeyStoreWrapperTest.kt]: the concept of mnemonic should not be known in this layer.

Comment (Daniel-Wang) [reply app/src/androidTest/java/blockeq/com/stellarwallet/encryption/KeyStoreWrapperTest.kt]: Yeah that's true, I'm not writing good tests XD

**Pull request merged**